admin/tabs/client/ClientTab.ts

```diff
@@ -7,18 +7,17 @@
 import {tabContainer} from '@xh/hoist/cmp/tab';
 import {hoistCmp} from '@xh/hoist/core';
 import {Icon} from '@xh/hoist/icon';
-import './ActivityTab.scss';
-import {clientErrorsPanel} from './clienterrors/ClientErrorsPanel';
+import {clientErrorsPanel} from '@xh/hoist/admin/tabs/client/errors/ClientErrorsPanel';
 import {feedbackPanel} from './feedback/FeedbackPanel';
-import {activityTrackingPanel} from './tracking/ActivityTrackingPanel';
+import {clientsPanel} from './clients/ClientsPanel';
 
-export const activityTab = hoistCmp.factory(() =>
+export const clientTab = hoistCmp.factory(() =>
     tabContainer({
         modelConfig: {
-            route: 'default.activity',
-            switcher: {orientation: 'left', testId: 'activity-tab-switcher'},
+            route: 'default.clients',
+            switcher: {orientation: 'left', testId: 'client-tab-switcher'},
             tabs: [
-                {id: 'tracking', icon: Icon.analytics(), content: activityTrackingPanel},
+                {id: 'clients', icon: Icon.desktop(), content: clientsPanel},
```

Comment:
`Connections` or something like that?

Reply:
I went ahead and updated the routes / display titles - I prefer them, happy to rename the components to match if you like them too (or leave those alone, but might as well be consistent)